posthog/warehouse/models/test/test_modeling.py

```diff
@@ -21,6 +21,7 @@
             {"events"},
         ),
         ("select 1", set()),
+        ("select * from numbers(10)", {"numbers"}),
```

Comment:
I think this may conflict with my earlier fix. But, by the way, this method will not fail on `numbers` as it just extracts the names, doesn't actually check that the table exists (see my earlier fix which includes `numbers`).

Reply:
Yeep - just resolved these

Reply:
Yeah i don't really understand how your test passed without my change 🤔

Reply:
Because the method I'm testing is not checking that the tables are valid names, just resolving the names.

Reply:
Ah, I was looking at the wrong method 🤦 I'll fix